### ISSUE TITLE
Migration failed

### ISSUE THREAD

**Issue by rafal6** [TheHive-Project/TheHive#2106]
### Request Type

I am trying to migrate from version 3.5 to 4.1.6. I have two VMs. Network traffic is released. Firewalls on the VMs are configured. Elasticsearch (TheHive 3.5.1) is configured to listen on 0.0.0.0. The new version of TheHive was set up and test run, everything worked.
Command to migrate:
```
/opt/thehive/bin/migrate --output /etc/thehive/application.conf --main-organisation test --es-uri http://10.10.10.1:9200 --es-index the_hive -d
```
The problem starts when I try to migrate, I get the message 'Migration failed'. Error dump below
```
[error] Migration failed
com.google.inject.ConfigurationException: Guice configuration errors:

1) No implementation for akka.actor.typed.ActorRef<org.thp.thehive.services.CaseNumberActor$Request> annotated with @com.google.inject.name.Named(value=case-number-actor) was bound.
  while locating akka.actor.typed.ActorRef<org.thp.thehive.services.CaseNumberActor$Request> annotated with @com.google.inject.name.Named(value=case-number-actor)
    for the 15th parameter of org.thp.thehive.services.CaseSrv.<init>(CaseSrv.scala:42)
  while locating org.thp.thehive.services.CaseSrv
    for the 4th parameter of org.thp.thehive.migration.th4.Output.<init>(Output.scala:90)
  while locating org.thp.thehive.migration.th4.Output

2) No implementation for akka.actor.typed.ActorRef<org.thp.thehive.services.CaseNumberActor$Request> annotated with @com.google.inject.name.Named(value=case-number-actor) was bound.
  while locating akka.actor.typed.ActorRef<org.thp.thehive.services.CaseNumberActor$Request> annotated with @com.google.inject.name.Named(value=case-number-actor)
    for the 15th parameter of org.thp.thehive.services.CaseSrv.<init>(CaseSrv.scala:42)
  while locating org.thp.thehive.services.CaseSrv
    for the 1st parameter of org.thp.thehive.connector.cortex.services.ActionOperationSrv.<init>(ActionOperationSrv.scala:20)
  while locating org.thp.thehive.connector.cortex.services.ActionOperationSrv
    for the 2nd parameter of org.thp.thehive.connector.cortex.services.ActionSrv.<init>(ActionSrv.scala:34)
  while locating org.thp.thehive.connector.cortex.services.ActionSrv
    for the 23rd parameter of org.thp.thehive.migration.th4.Output.<init>(Output.scala:90)
  while locating org.thp.thehive.migration.th4.Output

3) No implementation for akka.actor.typed.ActorRef<org.thp.thehive.services.CaseNumberActor$Request> annotated with @com.google.inject.name.Named(value=case-number-actor) was bound.
  while locating akka.actor.typed.ActorRef<org.thp.thehive.services.CaseNumberActor$Request> annotated with @com.google.inject.name.Named(value=case-number-actor)
    for the 15th parameter of org.thp.thehive.services.CaseSrv.<init>(CaseSrv.scala:42)
  while locating com.google.inject.Provider<org.thp.thehive.services.CaseSrv>
    for the 1st parameter of org.thp.thehive.services.TaskSrv.<init>(TaskSrv.scala:28)
  while locating org.thp.thehive.services.TaskSrv
    for the 1st parameter of org.thp.thehive.connector.cortex.services.EntityHelper.<init>(EntityHelper.scala:22)
  while locating org.thp.thehive.connector.cortex.services.EntityHelper
    for the 3rd parameter of org.thp.thehive.connector.cortex.services.ActionSrv.<init>(ActionSrv.scala:34)
  while locating org.thp.thehive.connector.cortex.services.ActionSrv
    for the 23rd parameter of org.thp.thehive.migration.th4.Output.<init>(Output.scala:90)
  while locating org.thp.thehive.migration.th4.Output

3 errors
        at com.google.inject.internal.InjectorImpl.getProvider(InjectorImpl.java:1120)
        at com.google.inject.internal.InjectorImpl.getProvider(InjectorImpl.java:1078)
        at com.google.inject.internal.InjectorImpl.getInstance(InjectorImpl.java:1131)
        at org.thp.thehive.migration.th4.Output$.apply(Output.scala:83)
        at org.thp.thehive.migration.Migrate$.$anonfun$new$1(Migrate.scala:198)
        at org.thp.thehive.migration.Migrate$.$anonfun$new$1$adapted(Migrate.scala:182)
        at scala.Option.foreach(Option.scala:407)
        at org.thp.thehive.migration.Migrate$.delayedEndpoint$org$thp$thehive$migration$Migrate$1(Migrate.scala:182)
        at org.thp.thehive.migration.Migrate$delayedInit$body.apply(Migrate.scala:16)
        at scala.Function0.apply$mcV$sp(Function0.scala:39)
```

### Work Environment
Old TheHive
| Question                   | Answer                                |
| -------------------------- | ------------------------------------- |
| OS version (server)        | CentOS  |
| OS version (client)        |         |
| Virtualized  Env.          | True                          |
| Dedicated RAM              | 16 GB                                 |
| vCPU                       |  8                       |
| TheHive version / git hash | 3.5.1               |
| Package Type               | RPM |
| Database                   | Elasticsearch           |

New TheHive
| Question                   | Answer                                |
| -------------------------- | ------------------------------------- |
| OS version (server)        | CentOS  |
| OS version (client)        |         |
| Virtualized  Env.          | True                          |
| Dedicated RAM              | 32 GB                                 |
| vCPU                       |  16                       |
| TheHive version / git hash | 4.1.6               |
| Package Type               | RPM |
| Database                   | Cassandra |
| Index type                 | Lucene                |
| Attachments storage        | Local               |
| Browser type & version     | If applicable                         |
### Question

Any suggestion what could be wrong?

**Comment by nadouani** [comment]
Hello, this is a duplicate of #2089 and will be fixed in 4.1.7. Thanks for reporting

**Comment by rafal6** [comment]
Ok thanks
